formats/calendar/SRF_Calendar.php

```diff
@@ -357,7 +367,12 @@ function displayCalendar( $events ) {
 		}
 
 		$cur_month = self::intToMonth( $cur_month_num );
-		$cur_year = date( 'Y', time() );
+
+		if ( is_numeric( $this->mStartYear ) && ( intval( $this->mStartYear ) == $this->mStartYear ) ) {
+			$cur_year = $this->mStartYear;
+		} else {
+			$cur_year = date( 'Y' );
```

Comment:
I'd say so.

Returns a string formatted according to the given format string using the given integer timestamp or the current time if no timestamp is given. In other words, timestamp is optional and defaults to the value of time(). [1]

[1] http://php.net/manual/en/function.date.php